src/Entity/LikesEntity.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Entity;
+
+class LikesEntity extends AbstractEntity
+{
+    public function __construct(
+        public readonly int $id,
+        public readonly string $username,
+        public readonly int $post_id,
+    )
+    {}
+
+    public function __get($id): ?int
```

Comment:
Please remove all those magic getter as those making code not really executable.